### ISSUE TITLE
My scripts involving dates no longer run with the update. 

### ISSUE THREAD

**Issue by kmaustral** [SilentVoid13/Templater#383]
**Plugin informations (please complete the following information):**
Windows 10
Obsidian 0.12.16
Templater 1.9.2

**Describe the bug**
When I attempt to insert a Templater script, nothing happens. There was no problem in the previous version. 

**Screenshots**
![image](https://user-images.githubusercontent.com/73867063/134995247-205ac8bf-e445-4de8-ab10-d596639fac4b.png)

**Comment by SilentVoid13** [comment]
Hey @kmaustral, this isn't a templater bug. Check #381 for more infos. You can now use the new hotkeys functionality built-in Templater if you want to bind a template to a hotkey.

**Comment by kmaustral** [comment]
Many thanks for this.  I keep getting this message when I try to run a
script:
Templater Error: Template parsing error, aborting.  Unexpected token '<'

I've uninstalled every plugin except Templater (including Hotkeys for
Templates). I can get a script to work first time, but then I get this
message whenever I want to run it again. I've downgraded to the previous
version, but it still has the same problem.

I realise it may not be an issue with Templater. But I'm truly
flummoxed about what's happening. Any thoughts would be much appreciated.

My best
Kevin


______________________

Melbourne, Narrm, Victoria, Australia

Poorneet 🦜 myrnong daisies flower, pied currawongs call

🌓 Last quarter

🗓️ Wednesday 29 September 02021

*🌡️ *Cloudy. Very high (95%) chance of showers, mainly during the
afternoon and evening. The chance of a thunderstorm, mainly about northern
suburbs in the late morning and afternoon. Winds easterly 15 to 20 km/h
turning southeasterly 20 to 30 km/h in the morning. 18℃

😷 Number of active coronavirus cases in Australia is 20,289, +159 from
yesterday. 6th lockdown in Narrm: day 56 of 79

🤔  "We need stories (and theories) that are just big enough to gather up
the complexities and keep the edges open and greedy for surprising new and
old connections." Donna Harraway

**Comment by kmaustral** [comment]
As an update, this script appeared in the Templater settings:
*obsidian.templater.block-ref.copy-alias.js*. I deleted it from the
Templater folder, and the other scripts then worked again.

There is one script not working after this still, which was before. I get
the error:

Templater Error: Template parsing error, aborting.
 finalResult is not defined
log_error @ Log.ts:17

I've attached the script in case that's of interest.

Much appreciated.
